Kernel/TTY/SlavePTY.cpp

```diff
@@ -80,7 +80,7 @@ void SlavePTY::on_master_write(UserOrKernelBuffer const& buffer, size_t size)
 
 ErrorOr<size_t> SlavePTY::on_tty_write(UserOrKernelBuffer const& data, size_t size)
 {
-    m_time_of_last_write = kgettimeofday().to_truncated_seconds();
+    m_time_of_last_write = kgettimeofday();
```

Comment:
(suspiciously lost)

Reply:
This is intentional. The change to use more precise time types in the Kernel was done some time ago by Andreas (even in a YouTube video), since many file systems and devices support sub-second precise timestamps unlike ext2's second precise time stamps. I just took this opportunity to increase SlavePTY's time stamp precision to nanoseconds.